src/components/UserSelector.tsx

```diff
@@ -1,16 +1,36 @@
-import React from 'react';
+import React, { useState } from 'react';
+import cn from 'classnames';
+
+import { User } from '../types';
+
+type Props = {
+  users: User[];
+  selectedUser: User | null;
+  onSelectUser: (user: User) => void;
+};
+
+export const UserSelector: React.FC<Props> = ({
+  users,
+  selectedUser,
+  onSelectUser,
+}) => {
+  const [isOpened, setIsOpened] = useState(false);
 
-export const UserSelector: React.FC = () => {
   return (
-    <div data-cy="UserSelector" className="dropdown is-active">
+    <div
+      data-cy="UserSelector"
+      className={cn('dropdown', { 'is-active': isOpened })}
+    >
       <div className="dropdown-trigger">
         <button
           type="button"
           className="button"
           aria-haspopup="true"
           aria-controls="dropdown-menu"
+          onBlur={() => setIsOpened(false)}
+          onClick={() => setIsOpened(current => !current)}
```

Comment:
Consider adding keyboard accessibility features, such as handling key events (e.g., 'Enter' or 'Escape') to open/close the dropdown. This will improve the component's usability for keyboard users.